src/ixdat/techniques/ms.py

```diff
@@ -1189,12 +1194,16 @@ def gas_flux_calibration_curve(
             )
         else:
             cal_type = "gas_flux_calibration_curve"
+            mol_conc_ppm = 10**6
```

Comment:
It seems this `mol_conc_ppm` never gets used!
Should it go into `mol_conc_ppm_list`?

Reply:
It was moved into the wrong place, should be fixed now.